source/brailleTables.py

```diff
@@ -557,10 +565,6 @@ def listTables():
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
 addTable("zh-tw.ctb", _("Chinese (Taiwan, Mandarin)"))

```

Comment:
please preserve the new line between these sections

```suggestion
addTable("zh-tw.ctb", _("Chinese (Taiwan, Mandarin)"))

```